cvat/apps/engine/task.py

```diff
@@ -387,11 +474,17 @@ def _create_thread(db_task, data, isBackupRestore=False, isDatasetImport=False):
     media = _count_files(data)
     media, task_mode = _validate_data(media, manifest_files)
 
+    if job_file_mapping is not None and task_mode != 'annotation':
```

Comment:
Should it be a part of _validate_job_file_mapping?

Reply:
The task mode is unknown prior to the previous line. The `_validate_job_file_mapping` call can be moved later, but it looks like it's better to fail as soon as possible in case of invalid parameters.